### ISSUE TITLE
想咨询下这个结果说明，感谢大神帮忙解答

### ISSUE THREAD

**Issue by dooozi** [kerlomz/captcha_trainer#23]
INFO:tensorflow:Step: 9200 Time: 13.979 sec/batch, Cost = 0.00025, 160x60-BatchSize: 128
INFO:tensorflow:Step: 9300 Time: 13.981 sec/batch, Cost = 0.00025, 160x60-BatchSize: 128
INFO:tensorflow:Step: 9400 Time: 13.941 sec/batch, Cost = 0.00025, 160x60-BatchSize: 128
INFO:tensorflow:Step: 9500 Time: 14.033 sec/batch, Cost = 0.00024, 160x60-BatchSize: 128
INFO:tensorflow:Step: 9600 Time: 14.009 sec/batch, Cost = 0.00024, 160x60-BatchSize: 128
INFO:tensorflow:Step: 9700 Time: 13.956 sec/batch, Cost = 0.00024, 160x60-BatchSize: 128
INFO:tensorflow:Step: 9800 Time: 14.029 sec/batch, Cost = 0.00023, 160x60-BatchSize: 128
INFO:tensorflow:Step: 9900 Time: 14.055 sec/batch, Cost = 0.00023, 160x60-BatchSize: 128
INFO:tensorflow:Step: 10000 Time: 14.234 sec/batch, Cost = 0.00023, 160x60-BatchSize: 128
INFO:tensorflow:0 7 7 [13, 11, 26, 30, 13, 18, 11] [13, 11, 26, 30, 13, 18, 11] --> ['c', 'a', 'p', 't', 'c', 'h', 'a'] ['c', 'a', 'p', 't', 'c', 'h', 'a']
INFO:tensorflow:1 7 7 [13, 11, 26, 30, 13, 18, 11] [13, 11, 26, 30, 13, 18, 11] --> ['c', 'a', 'p', 't', 'c', 'h', 'a'] ['c', 'a', 'p', 't', 'c', 'h', 'a']
INFO:tensorflow:2 7 7 [13, 11, 26, 30, 13, 18, 11] [13, 11, 26, 30, 13, 18, 11] --> ['c', 'a', 'p', 't', 'c', 'h', 'a'] ['c', 'a', 'p', 't', 'c', 'h', 'a']
INFO:tensorflow:3 7 7 [13, 11, 26, 30, 13, 18, 11] [13, 11, 26, 30, 13, 18, 11] --> ['c', 'a', 'p', 't', 'c', 'h', 'a'] ['c', 'a', 'p', 't', 'c', 'h', 'a']
INFO:tensorflow:4 7 7 [13, 11, 26, 30, 13, 18, 11] [13, 11, 26, 30, 13, 18, 11] --> ['c', 'a', 'p', 't', 'c', 'h', 'a'] ['c', 'a', 'p', 't', 'c', 'h', 'a']
ERROR:tensorflow:[]
INFO:tensorflow:Epoch: 1358, Step: 10000, Accuracy = 1.0000, Cost = 0.00023, Time = 15.367 sec/batch, LearningRate: 0.0009800000116229057
INFO:tensorflow:CNN Output: (?, ?, 256)
INFO:tensorflow:./model/myfirsttest.model-10000
INFO:tensorflow:Restoring parameters from ./model/myfirsttest.model-10000
INFO:tensorflow:Froze 40 variables.
Converted 40 variables to const ops.
INFO:tensorflow:Total Time: 33.93611812591553 sec.
INFO:tensorflow:Training completed.

**Comment by dooozi** [comment]
感觉自己完全 看不懂

**Comment by kerlomz** [comment]
Google Translate了解一下，或者其他翻译软件。
